dotcom-rendering/src/components/FollowButtons.tsx

```diff
@@ -92,9 +92,14 @@ type ButtonProps = {
 export const FollowNotificationsButton = ({
 	isFollowing,
 	onClickHandler,
-}: ButtonProps) => {
+	cssOverrides,
+}: ButtonProps & { cssOverrides?: SerializedStyles }) => {
```

Comment:
We generally consider `cssOverrides` to be an anti-pattern, as it breaks down the ability to be confident in how a component looks.

It would be preferable to have a dedicated `withExtraBottomMargin` boolean or similar.